CONTRIBUTING.md

```diff
@@ -610,6 +610,25 @@ Then put this:
 export NODE_OPTIONS=--no-experimental-fetch
 ```
 
+If Following type of error comes while running dev server(i.e using above commands):
+
+```bash
+Error: ENOSPC: System limit for number of file watchers reached
+```
+The meaning of this error is that the number of files monitored by the system has reached the limit!
+
+Do the following to Modify the number of system monitoring files :
+```bash
+sudo gedit /etc/sysctl.conf
+```
+Add a line at the bottom
+```bash
+fs.inotify.max_user_watches=524288
```

Comment:
How do you decide this value? Please add commands to get the current value. 
It's worth adding a comment asking to `identify current value and make a decision based on that`. 

Also, Please mention that value needs to be decided based on the system memory: https://stackoverflow.com/questions/535768/what-is-a-reasonable-amount-of-inotify-watches-with-linux

Reply:
I added a link to this StackOverflow post as a suggestion above - thanks for finding it!